src/stateless/DefaultListItem.vue

```diff
@@ -185,7 +185,7 @@ export default {
         &__label {
             color: @gunpowder;
             &--selected { color: @royal-blue; }
-            &--disabled { color: @gray-blue; }
+            &--disabled { color: @very-light-gray; }
```

Comment:
Why this change? I'm a bit worried this might break sth since default list item is used on a lot of places

Reply:
Uh actually we don't need this now, this wouldn't change anything. Would have to add different colors on hover so it would make a difference.

Input is kinda strange now, for label text we have ```@gunpowder -> black``` on hover while the text entered in input is always black. This looks kinda strange next to selectboxes which have ```@gunpowder -> black``` both is selected or "Please select" state. We actually have it that way in designs, but should probably unify with other elements

Reply:
commented on wrong issue...

Reply:
This is for disabled selectbox state, we should probably override it in ```selectbox.vue```instead yeah

Reply:
Yes, this is a bit of a mess now with the whole 'inheritance' structure, we need to talk about how will we resolve this but for now an override would probably be better.